go/consensus/tendermint/apps/registry/transactions.go

```diff
@@ -540,20 +540,6 @@ func (app *registryApplication) registerRuntime( // nolint: gocyclo
 		return registry.ErrIncorrectTxSigner
 	}
 
-	// If TEE is required, check if runtime provided at least one enclave ID.
```

Comment:
This was moved into `VerifyRegisterRuntimeArgs` since currently it was skipped by sanity checks and only checked on the register transactions.